Tribler/Core/TorrentDef.py

```diff
@@ -128,26 +126,22 @@ def _create(metainfo):  # TODO: replace with constructor
     _create = staticmethod(_create)
 
     @staticmethod
+    @blocking_call_on_reactor_thread
     def load_from_url(url):
         """
-        If the URL starts with 'http:' load a BT .torrent or Tribler .tstream
-        file from the URL and convert it into a TorrentDef. If the URL starts
-        with our URL scheme, we convert the URL to a URL-compatible TorrentDef.
-
-        If we can't download the .torrent file, this method returns None.
+        Load a BT .torrent or Tribler .tstream file from the URL and
```

Comment:
Do we still support `.tstream` files? If not, maybe this part of the comment can be removed.